cypress/integration/group1/dropdown.ts

```diff
@@ -77,6 +77,45 @@ describe('Dropdown test', () => {
     it('Should show all accounts as linked (except GitLab and Bitbucket)', () => {
       everythingOk();
     });
+    setTokenUserViewPortCurator();
+    // Check that changing the tab changes the url
+    it('should default to accounts tab', () => {
+      cy.visit('/accounts');
+      cy.url().should('eq', Cypress.config().baseUrl + '/accounts?tab=accounts');
+    });
+    it('should default to accounts tab when tab name misspelled', () => {
+      cy.visit('/accounts?tab=abcd');
+      cy.url().should('eq', Cypress.config().baseUrl + '/accounts?tab=accounts');
+    });
+    it('Change tab to profiles', () => {
+      goToTab('Profiles');
+      cy.url().should('eq', Cypress.config().baseUrl + '/accounts?tab=profiles');
+    });
+    it('Change tab to dockstore account controls', () => {
+      goToTab('Dockstore Account Controls');
+      cy.url().should('eq', Cypress.config().baseUrl + '/accounts?tab=dockstore%20account%20controls');
+    });
+    it('Change tab to requests', () => {
+      goToTab('Requests');
+      cy.url().should('eq', Cypress.config().baseUrl + '/accounts?tab=requests');
+    });
+    // Check that changing the url changes the tab
```

Comment:
This doesn't really test whether the tab is active.  Should have:
```
getTab(thing).parent().should('have.class', 'mat-tab-label-active');
```

Reply:
Keep the url check for the first four tests?

Reply:
I think URL checks are good, a combination of both checks is ideal.